### ISSUE TITLE
Hyperledger Benchmarking Issues

### ISSUE THREAD

**Issue by BGerard** [ooibc88/blockbench#12]
Hi guys,
I'm attempting to use Blockbench as part of my final third year university project, however, i'm having trouble configuring it. I have run the install (built peer), built the macro drivers, installed the micro dependencies via npm, patched, built and installed restclient-cpp. Now I am trying to run start-root and start-peer to try and actually run some benchmarks and this is where I run into trouble. 

env.sh 
https://pastebin.com/CiGZeSU5

hl_log (as created by start-root)
https://pastebin.com/dY4cJtSq

If someone could provide some help it would be greatly appreciated.

**Comment by ug93tad** [comment]
did you use Hyperledger v0.6 version? 
is there a consensus/pbft/config.yaml file in your Hyperledger installation directory?

**Comment by BGerard** [comment]
So my go install is in /usr/lib/go-1.6.
Inside here I then have a link to the github.com folder within the src folder of my data directory (this contains the built peer etc), this also contains consensus/pbft/config.yaml.
Hyperledger version: 0.6.1-preview

Also, the hosts within the host files, I need to create them? I.e: set up a cluster of virtual machines on a private network with no ssh authentication and then add their ips to the hosts file? Or does blockbench do some stuff with docker that does this for me?

Thanks for the response

**Comment by ug93tad** [comment]
yes, a cluster is needed in which each node has its own Hyperledger source directory.

**Comment by BGerard** [comment]
So i'd be good to take the vm I already have an clone it, have all the dependencies etc then?

**Comment by xiaohua0921** [comment]
I checked the main.go file and found that there are missing three dependent libraries , sys ,text  and viper , 
1、we should mkdir /golang.org/x under the directory of src . 
2、then    go get https://github.com/golang/sys.git ;    go get https://github.com/spf13/cobra  ;   go get https://github.com/spf13/viper
3、We need to pay attention to the 215 line in core.yaml，should modify the path.
4、sudo mkdir /var/hyperledger 
     sudo chmod  777 /var/hyperledger
5、modify the 16 line in start-peer.sh into “ssh  $peer ". ~/.profile && . $HL_HOME/start-root.sh"  ”
